Add more context for buck users on the coreml pte2 quantization errors

derekxu opened the pull request:
Summary:
Add more context for buck users on the coreml pte2 quantization errors
- See D68569845 for details

Differential Revision: D68569845

derekxu commented:
@pytorchbot label "topic: not user facing"

cccclai reviewed (approved):
Thanks - are you linking the wrong diff in the summary?

Commit:
Add more context for buck users on the coreml pte2 quantization errors (#7891)

Summary:

Add more context for buck users on the coreml pte2 quantization errors
- See D68522589 for details and context

Reviewed By: cccclai

Differential Revision: D68569845
